frontend/src/components/elements/CodeBlock/CodeBlock.tsx

```diff
@@ -25,6 +25,11 @@ import {
   StyledCopyButtonContainer,
 } from "./styled-components"
 
+import {
+  CheckIfChildrenIncludesValidColors,
+  PrepareColoredSpanJSXElements,
+} from "../../shared/StreamlitMarkdown/StreamlitMarkdown"
```

Comment:
nit: I think we usually use the full path for importing if it is not within the same folder:

```typescript
"src/components/shared/StreamlitMarkdown/StreamlitMarkdown"
```

Reply:
Thank you, it's updated now 👍